scheduler/scheduler.go

```diff
@@ -274,9 +279,14 @@ func findAvailableParent(d distro.Distro) (host.Host, error) {
 
 // numNewParentsNeeded returns the number of additional parents needed to
 // accommodate new containers
-func numNewParentsNeeded(numCurrentParents, numContainersNeeded, numExistingContainers, maxContainers int) int {
+func numNewParentsNeeded(numCurrentParents, numUninitializedParents, numContainersNeeded, numExistingContainers, maxContainers int) int {
```

Comment:
does it make sense to change the arguments to this function into a struct so you can get keyword-ish arguments? it seems really easy to flip a couple of arguments and get really weird responses.